geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/TiledRasterRDD.scala

```diff
@@ -429,6 +436,25 @@ class SpatialTiledRasterRDD(
     SpatialTiledRasterRDD(None, multibandRDD)
   }
 
+  def hillshade(
+    sc: SparkContext, 
```

Comment:
In which case we should pull it out of the costDistance parameter list as well, since that was my template